lib/ecl/ecl_sum_data.cpp

```diff
@@ -205,12 +210,8 @@
 
 
 
-#define INVALID_MINISTEP_NR -1
-#define INVALID_TIME_T 0
-
 struct ecl_sum_data_struct {
   ecl_smspec_type        * smspec;                 /* A shared reference - only used for providing good error messages. */
-  vector_type            * data;                   /* Vector of ecl_sum_tstep_type instances. */
   double                   days_start;
```

Comment:
It seems several of these fields are now duplicated between this struct and the new  ecl_sum_file_data class?